### ISSUE TITLE
Conversation without messages

### ISSUE THREAD

**Issue by hartois** [chatwoot/chatwoot#1653]
If conversation does not have messages (created over API) we have JS error in UI

**To Reproduce**

Create conversation, but does not send messages
Open JS console
Open Inbox

Inbox will be empty, error in JS-console

**Expected behavior**

Listing empty conversations

**Screenshots**

![image](https://user-images.githubusercontent.com/11641656/104629056-0861fa80-56aa-11eb-9e0c-475cf7b11e3b.png)
![image](https://user-images.githubusercontent.com/11641656/104629066-0ac45480-56aa-11eb-9bec-cc28b5457155.png)
![image](https://user-images.githubusercontent.com/11641656/104629075-0d26ae80-56aa-11eb-9f08-ed4bf2ad0645.png)

**Environment**

Self hosted, v1.11.1

**Comment by pranavrajs** [comment]
@hartois Any idea how you got a conversation without a message. Do you use API Channel?

**Comment by hartois** [comment]
@pranavrajs, yes, API Channel.
```
POST {"source_id": "<source_id from creating contact>"} to  /accounts/1/conversations
```

**Comment by pranavrajs** [comment]
@hartois Sorry, I didn't see `created over API` part, we need to filter out the conversation without messages.

**Comment by hartois** [comment]
May be.
But why can't there be a conversation without messages? So to say "preliminary"...

**Comment by hartois** [comment]
The controller should not return "null" anyway as an element of the messages array

**Comment by pranavrajs** [comment]
@hartois True! I agree.

**Comment by pranavrajs** [comment]
Closed via #1683
